pmrmoura/TeamGC
# fix(UI): navigation

### Issue Name
**What I did:**

- Fix onboarding navigation to show just only when launch at the first time;

- Fix back button navigation in screens.

**How to test:**

- Check if the back button was removed from the SearchView and WaitingView.

-  Check if when you open the application for the second time, the onboarding will show up.
